src/controllers/main/main.ts

```diff
@@ -1708,12 +1721,20 @@ export class MainController extends EventEmitter {
   #throwBroadcastAccountOp({ message: _msg, error: _err }: { message?: string; error?: Error }) {
     let message = _msg || `Unable to broadcast the transaction. ${_err?.message || 'unknown'}`
 
-    // Enhance the error incoming for this corner case
-    if (message.includes('insufficient funds'))
-      message = 'Insufficient funds to cover the fee for broadcasting the transaction.'
-
-    // Trip the error message, errors coming from the RPC can be huuuuuge
-    message = message.length > 300 ? `${message.substring(0, 300)}...` : message
+    if (message) {
+      if (message.includes('insufficient funds')) {
+        // TODO: Better message?
+        message = 'Insufficient funds for intristic transaction cost'
```

Comment:
yea, we need a better message. who doesn't have enough funds? you don't have enough native?